src/dsp_tools/xmllib/helpers.py

```diff
@@ -210,15 +203,9 @@ def find_date_in_string(string: str) -> str | None:
         DSP-formatted date string, or None
 
     Examples:
-        ```python
-        result = xmllib.find_date_in_string("1849/1850")
-        # -> "GREGORIAN:CE:1849:CE:1850"
-        ```
-
-        ```python
-        result = xmllib.find_date_in_string("not a valid date")
-        # -> None
-        ```
+        >>> xmllib.find_date_in_string("1849/1850")
+        'GREGORIAN:CE:1849:CE:1850'
+        >>> xmllib.find_date_in_string("not a valid date")
```

Comment:
I haven't found out a nice way how to display "no output".

Reply:
`None` ?